scRNA-seq-advanced/04-differential_expression.Rmd

```diff
@@ -0,0 +1,544 @@
+*Need to explain the difference between tumor/tumor_myoblast, etc*
+These plots show that ARMS and ERMS share a lot of the same cell types, but perhaps there are differences in the cells in each sub type.
+We also see that 6 of the libraries show a mixture of tumor cells belonging to the mesoderm, myoblast, and myocyte classes.
+3 libraries contain cells that are only classified as tumor or non-tumor, and tumor cells are not further classified and the remaining library appears to not have been assigned any cell types, as they are all labeled with `NA`. 
+
+```{r}
+# define libraries to keep
+library_ids <- c(
+  "SCPCL000479",
+  "SCPCL000480",
+  "SCPCL000481",
+  "SCPCL000484",
+  "SCPCL000488",
+  "SCPCL000491"
+)
+
+# subset sce to only contain samples with library IDs of interest 
+samples_to_keep <- integrated_sce$sample %in% library_ids
+rms_sce <- integrated_sce[,samples_to_keep]
+```
+
+```{r}
+# we are left with 3 from each sub type 
+sample_metadata %>%
+  dplyr::filter(library_id %in% library_ids) %>%
+  dplyr::pull(subdiagnosis) %>%
+  table()
+```
+
+
+```{r}
+# look at distribution of cell types across all libraries 
+as.data.frame(colData(rms_sce)) %>%
+  dplyr::count(celltype_broad)
+```
+
+The `Tumor_Myoblast` cell type is the most prominent cell type, and based on the UMAPs is found in all 6 of our samples.
+We will start by comparing the gene expression of the myoblasts across ARMS and ERMS samples.
+
+
+## Differential Expression Analysis
+
+### Pseudo-bulking 
+
+Before we can compare the gene expression profiles of myoblasts in ARMS vs. ERMS patients, we will need to "pseudo-bulk" the gene counts. 
+Pseudo-bulking creates a new counts matrix that contains the total counts across all cells of a given label (e.g. cell type) for each sample. 
+This allows us to use single-cell resolution to define the labels and sum gene counts across groups of cells containing the same label, and avoids counting each cell as its own replicate. 
+
+Pseudo-bulking is implemented prior to differential expression analysis for the following reasons: 
+
+- Produces larger counts, which are more amenable to standard normalization and differential expression methods used by bulk RNA-sequencing. 
+-  Collapses gene expression counts by sample, so that samples, rather than cells, represent replicates.
+- Masks variance within a sample to highlight variance across samples.
+
+We are staring with the simple comparison of looking at one cell type and comparing across two groups of samples, samples belonging to either the ERMS and ARMS subtypes. 
+When we do this, we will first want to pseudo-bulk our dataset to group our cells by cell type and by sample.
+This will create a new `SingleCellExperiment` object that contains the pseudo-bulked counts to be used as input for differential expression analysis. 
+Then we can subset the pseudo-bulked object to just contain our cell type of interest before proceeding. 
```

Comment:
```suggestion
This will create a new `SingleCellExperiment` (SCE) object that contains the pseudo-bulked counts for all genes. 
We can subset this SCE to just contain our cell type of interest (tumor myoblasts) for input to differential expression analysis. 
```